Add postmortem feature updates (#36690)
DataDog/documentation: content/en/incident_response/incident_management/setup_and_configuration/templates.md

## Postmortems

Postmortem templates define the structure and content of your postmortems. When you generate a postmortem, Datadog automatically populates the template with data from the incident. Use [template variables][4], such as `{{incident.severity}}`, to control which incident data is included.

To create a postmortem template:

> 1. Go to [**Incident Management Settings > Post-Mortem Templates**][8] and click **+ New Postmortem Template**.

Reviewer: ```suggestion
1. Go to [**Incident Management Settings > Postmortem Templates**][8] and click **+ New Postmortem Template**.
```

{{% collapse-content title="Standard variables" level="h4" %}}
Standard variables pull directly from incident fields and also include any defined [custom property fields][7].

| Variable | Description |
|---|---|
| `{{incident.title}}` | The incident title |
| `{{incident.state}}` | The current incident state (for example: `active`, `stable`, `resolved`) |
| `{{incident.severity}}` | The incident severity level |
| `{{incident.commander}}` | The incident commander's name, or their email or handle if no name is set |

{{% /collapse-content %}}

{{% collapse-content title="Incident card variable" level="h4" %}}
The incident card variable (`{{incident.card}}`) embeds a summary card that reflects incident fields in real time. When incident fields change, the card in the postmortem updates automatically. The card includes postmortem status and owner fields.

**Note**: The incident card is only supported in Datadog Notebooks.
{{% /collapse-content %}}

{{% collapse-content title="Follow-ups variable" level="h4" %}}

> The follow-ups variable (`{{incident.follow-ups}}`) embeds a list of incident follow-ups in the postmortem notebook. From this view, you can assign follow-ups, set due dates, and create new ones. Changes are reflected in the incident's follow-ups list.

Reviewer: ```suggestion
The follow-ups variable (`{{incident.follow-ups}}`) embeds a list of incident follow-ups in the postmortem notebook. From the list view in your notebook, you can set due dates, assign items, or create new follow-up items. Changes are reflected in the incident's follow-ups list.
```
